Application/EdFi.Ods.Admin/Services/SecurityService.cs

```diff
@@ -41,46 +45,38 @@ public static UserLookupResult ForUser(User user)
         }
     }
 
-    public class UserIdLookupResult
-    {
-        public static readonly UserIdLookupResult Empty = new UserIdLookupResult();
-
-        public int CurrentUserId { get; private set; }
-
-        public bool HasCurrentUser
-        {
-            get { return CurrentUserId != default(int); }
-        }
-
-        public static UserIdLookupResult ForUserId(int id)
-        {
-            return new UserIdLookupResult
-                   {
-                       CurrentUserId = id
-                   };
-        }
-    }
 
     public class SecurityService : ISecurityService
     {
         private readonly IClientAppRepo _clientAppRepo;
+        private readonly IIdentityProvider _identityProvider;
 
-        public SecurityService(IClientAppRepo clientAppRepo)
+        public SecurityService(IClientAppRepo clientAppRepo, IIdentityProvider identityProvidert)
         {
             _clientAppRepo = clientAppRepo;
+            _identityProvider = identityProvidert;
         }
 
         public UserLookupResult GetCurrentUser()
         {
             try
             {
-                var idLookupResult = GetCurrentUserId();
+                string currentUserName = HttpContext.Current.User?.Identity?.Name;
```

Comment:
should we push down the user from the http context instead as a dependency for the method?